docs/howtos/locally-published-components-in-fenix.md

```diff
@@ -25,19 +25,18 @@ in their build. The workflow is:
 1. Build the consuming project following its usual build procedure, e.g. via `./gradlew assembleDebug` or `./gradlew
    test`.
 
-If all goes well, this should automatically build your checkout of application-servies, publish it
-to a local maven repository, and configure the consuming project to install from there instead of
+If all goes well, this should automatically build your checkout of `application-services`, publish it
+to a local maven repository, and configure the consuming project to install it from there instead of
 from our published releases.
 
 ## Using a manual workflow
```

Comment:
The autopublish system has been working pretty well for me, maybe we don't need to include the how to do this manually if the code breaks part in the HOWTO.  I think it could live in the design directory instead.  The same goes for the "how to cut a release" doc.

Feel free to ignore this one though, there's already plenty of changes in the PR.

Reply:
I'll do this in a follow-up, I have one follow-up coming up to restructure our docs directors to reflect the structure defined in the `SUMMARY.md` and I'll squeeze this in!